crates/sui-indexer-alt/migrations/2024-10-21-003426_tx_indices/up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS tx_affected_addresses (
+    tx_sequence_number          BIGINT       NOT NULL,
+    affected                    BYTEA        NOT NULL,
+    sender                      BYTEA        NOT NULL,
+    PRIMARY KEY(affected, tx_sequence_number)
+);
+
+CREATE INDEX IF NOT EXISTS tx_affected_addresses_tx_sequence_number_index ON tx_affected_addresses (tx_sequence_number);
+CREATE INDEX IF NOT EXISTS tx_affected_addresses_sender ON tx_affected_addresses (sender, affected, tx_sequence_number);
+
+CREATE TABLE IF NOT EXISTS tx_digests (
+    tx_digest                   BYTEA        PRIMARY KEY,
+    tx_sequence_number          BIGINT       NOT NULL
+);
+CREATE INDEX IF NOT EXISTS tx_digests_tx_sequence_number_index ON tx_digests (tx_sequence_number);
+
+CREATE TABLE IF NOT EXISTS tx_kinds (
+    tx_sequence_number          BIGINT       NOT NULL,
+    tx_kind                     SMALLINT     NOT NULL,
+    PRIMARY KEY(tx_kind, tx_sequence_number)
```

Comment:
It's a bit non-intuitive when the order of the columns don't match the order of primary key.